tests/test_immutablemultidict.py

```diff
@@ -150,11 +150,11 @@ def test_of(self):
 
     def test_repr(self):
         self.assertEqual(
-            "i{1: [2, 2, 3]}", repr(ImmutableListMultiDict.of({1: [2, 2, 3]}))
+            "i{1: (2, 2, 3)}", repr(ImmutableListMultiDict.of({1: [2, 2, 3]}))
```

Comment:
A decision needs to be made here:
- Keeping the `[...]` might be misleading, even if the `repr` is still valid for initialization, unless the fact that a tuple is backing the multidict is an implementation detail that users don't need to know about. This would also be in line with Guava.
- The class could reflect the implementation and be renamed to `ImmutableTupleMultiDict`.

Reply:
I'm indecisive on all counts. @rgabbard, would love to hear your thoughts. I think it does not need to be renamed and that the repr with `(` is ideal since it's a little clearer (even if the backing structure we not a tuple, I still like the "this is not a Python `list`" hint.

Reply:
My inclination is the same as @ConstantineLignos : to keep the `ImmutableListMultiDict` name (for Guava parallelism, and also just because renaming frequently-used things is a pain) but switch to using `(` in the `repr`.

Reply:
In that case, it will remain as-is.